documentation-site/pages/components/tag.mdx

```diff
@@ -35,20 +37,28 @@ Compact UI to display a collection of consice information.
   <Basic />
 </Example>
 
-<Example title="Tag with all available kinds" path="examples/tag/kinds.js">
+<Example title="Tags of all available kinds (closable/non-clickable)" path="examples/tag/kinds.js">
```

Comment:
Should we be more descriptive on what are these examples? Maybe something like

`Tags of all available closable kinds, with non-clickable body`?